README.org

```diff
@@ -82,15 +84,25 @@ Stylus can be very confusing when it comes to importing/requiring sheets into ot
 This is how the sheets ~require~ in this project:
 
 + =Makefile=
-    - =styl/{dark,light}.styl= (in Makefile syntax: ~$$color.styl~)
-        + =colors=
-        + Contents of ={dark,light}.styl=
+    - =theme=: contains custom theme files. Each theme corresponds to a group of css files.
+        - =*/{dark,light}.styl= (in Makefile syntax: ~$$color.styl~)
+            - =colors=
+            - Contents of ={dark,light}.styl=
     - =styl= (which loads =styl/index.styl=)
-        + =generic=
-            - =mixins=
+        - =mixins=
     - =sites/*.styl=: The site-specific sheets, as well as =generic.styl=, which applies to all of them, and also builds as a separate sheet for non-specific sites.  =all-sites.styl= puts all of the site-specific sheets into one big CSS file, which some people may prefer over setting up custom CSS for each site in their browser.
 
-This way, the Solarized color values are loaded first, followed by the dark- or light-themed mappings of those color values to friendly names, after which those friendly names can be used in the sheets that actually style elements and pages.
+This way, the color values are loaded first, followed by the dark- or light-themed mappings of those color values to friendly names, after which those friendly names can be used in the sheets that actually style elements and pages.
+
+*** New Themes
```

Comment:
Thanks for adding this section.  I think we could get by with it being even simpler, maybe something like:

```
It's easy to add your own themes:

1.  Copy an existing theme directory in the =themes/= directory, giving it a new name.
2.  Modify it as required.
3.  Run `make` in the project root directory to build the CSS files.
```